packages/grafana-ui/src/components/Switch/Switch.tsx

```diff
@@ -27,8 +27,10 @@ export const Switch = forwardRef<HTMLInputElement, Props>(
       <div className={cx(styles.switch, invalid && styles.invalid)}>
         <input
           type="checkbox"
+          role="switch"
```

Comment:
We are relying on the `checkbox` role in all many tests using this component, so all those tests would need to be updated as well.

Reply:
Hi Alex. Thank you for taking a look at the PR. As far as I could see, there was only one test affected by this change, and that is in the `AutoSaveField` file. Am I missing something?

Reply:
Strange, I thought there'd be more. Have you also checked grafana-enterprise, which uses this component?

Reply:
I have only checked the `grafana-ui` package here. :-(

Indeed in the `cloud-onboarding/plugin` where we use this component, we would also need to update the tests to check by "switch" instead of "checked". 

As far as I understand the steps to take after merge of this PR would be:
- release new grafana-ui version
- updates the version in the repos where this package is used and updates the tests in all the components that use it. 

Is that the process?

**And could you link the `grafana-business` repo please?**

Reply:
> Indeed in the cloud-onboarding/plugin where we use this component, we would also need to update the tests to check by "switch" instead of "checked".

Another issue with this is that we also have 3rd-party plugins which could be affected by this change, so this could be possible considered a breaking change. I'm. not sure how big of a deal this is, as it only affects the tests, but this least should be added to the changelog. 

> As far as I understand the steps to take after merging this PR would be:

> release new grafana-ui version
> update the version in the repos where this package is used and update the tests in all the components that use it

grafana/ui is released together with grafana, there's no separate releases for it. This means the release window can be pretty long.

Reply:
> As far as I could see, there was only one test affected by this change, and that is in the AutoSaveField file. Am I missing something?

If you check the build, specifically the failing [test](https://drone.grafana.net/grafana/grafana/186974/2/6) step, you can see all the tests that fail due to this change.

Reply:
I agree Alex, that this being a breaking change and all, it is not worth us risking it. As explained in the PR, the screen readers properly render the "checkbox", so I would not continue this path. ☺️